events/static/events/js/script.js

```diff
@@ -0,0 +1,71 @@
+async function initMap() {
+    map = new google.maps.Map(document.getElementById("map"), {
+      center: { lat: 40.71159438471519, lng: -73.97424027294727 },
+      zoom: 12,
+      mapId: '9a7e26117f10196d'
+    });
+        const responseEvent = await fetch('get_data/');
+        const responseLocation = await fetch('get_locations/');
+        const event_data = await responseEvent.json();
+        const location_objects = await responseLocation.json();
+        const location_data = event_data.location_data;
+        const locations = location_objects.locations;
+        const locationmap = new Map();
+        for(let i =0; i<locations.length;i++){
+            locationmap.set(locations[i].pk,[locations[i].fields.latitude,locations[i].fields.longitude])
+        }
+        let duplicateLocationsMap = {};
+        for(let i=0; i<location_data.length;i++){
+            let loc = location_data[i].fields;
+            if(loc.is_active==true){
```

Comment:
y not just fetch active data?  Fetching inactive data will lead to a huge number of response objects

Reply:
Okay makes sense

Reply:
Change has been made. Please take a look